ghcide/src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -219,11 +225,23 @@ atPoint IdeOptions{} (HAR _ hf _ _ kind) (DKMap dm km) pos = listToMaybe $ point
         prettyName (Right n, dets) = T.unlines $
           wrapHaskell (showNameWithoutUniques n <> maybe "" (" :: " <>) ((prettyType <$> identType dets) <|> maybeKind))
           : definedAt n
+          ++ maybeToList (prettyPackageName n)
           ++ catMaybes [ T.unlines . spanDocToMarkdown <$> lookupNameEnv dm n
                        ]
           where maybeKind = fmap showGhc $ safeTyThingType =<< lookupNameEnv km n
         prettyName (Left m,_) = showGhc m
 
+        prettyPackageName n = do
+          m <- nameModule_maybe n
+          let pid = moduleUnitId m
+          conf <- lookupPackage unsafeGlobalDynFlags pid
+          let pkgName = T.pack $ packageNameString conf
+              version = T.pack $ showVersion (packageVersion conf)
+              libName = case sourceLibName conf of
+                Just (PackageName x) -> ":" <> T.pack (unpackFS x)
+                _                    -> ""
+          pure $ " *(" <> pkgName <> "-" <> version <> libName <> ")*"
+
```

Comment:
Could you add a test for this? Like whether a function contains its library name or not